media/js/base/global.js

```diff
@@ -190,3 +190,45 @@ function gaTrack(eventArray, callback) {
         }
     }
 }
+
+
+// Retrieve search params as a object for easier access
+// This is a simple version of https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+var _SearchParams = function (search) {
+    var params = this.params = {};
+
+    search = search || location.search || '';
+    search = search.match(/^\??(.*)/)[1];
+    search = search ? search.split(/[&;]/m) : [];
+
+    $.each(search, function (index, param) {
+        param = param.split('=');
+
+        var key = param[0];
+        var value = param[1]
+
+        params[key] = !isNaN(value) ? parseInt(value) : value;
```

Comment:
[`parseFloat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) might be better..